cvat-sdk/cvat_sdk/core/uploading.py

```diff
@@ -13,21 +13,130 @@
 import urllib3
 
 from cvat_sdk.api_client.api_client import ApiClient, Endpoint
+from cvat_sdk.api_client.exceptions import ApiException
 from cvat_sdk.api_client.rest import RESTClientObject
 from cvat_sdk.core.helpers import StreamWithProgress, expect_status
 from cvat_sdk.core.progress import NullProgressReporter, ProgressReporter
 
 if TYPE_CHECKING:
     from cvat_sdk.core.client import Client
 
+import tusclient.uploader as tus_uploader
+from tusclient.client import TusClient
+from tusclient.client import Uploader as TusUploader
+from tusclient.request import TusRequest, TusUploadFailed
```

Comment:
```suggestion
from tusclient.client import _TusClient
from tusclient.client import Uploader as _TusUploader
from tusclient.request import _TusRequest, _TusUploadFailed
```

Reply:
That doesn't make sense - there's no `_TusClient` class, only `TusClient`. Do you mean something like `from tusclient.client import TusClient as _TusClient`?

Reply:
Yes

Reply:
Okay, done.